app/src/layers/layersActions.js

```diff
@@ -126,9 +128,18 @@ export function initLayers(workspaceLayers, libraryLayers) {
       .forEach((heatmapLayer) => {
         const headerPromise = loadLayerHeader(heatmapLayer.url, getState().user.token);
         headerPromise.then((header) => {
+          const defaultFilter = {
```

Comment:
isn't that the fallback hack we wanna remove? please document it, or we may think it's "real code"

Reply:
Yes, this should be documented